lib/ios-sim.ts

```diff
@@ -14,3 +14,39 @@ var fiber = Fiber(() => {
 });
 
 fiber.run();
+
+global.publicApi = {};
+
+Object.defineProperty(global.publicApi, "getRunningSimulator", {
+	get: () => {
+		return (...args: any[]) => {
+			let future = new Future<any>();
+			let timer = setInterval(() => {
+				Fiber(() => {
+					let libraryPath = require("./iphone-simulator-xcode-7");
+					let obj = new libraryPath.XCode7Simulator()
+					let result = obj.getBootedDevice.apply(obj, args).wait();
+					if(result && !future.isResolved()) {
+						clearInterval(timer);
+						future.return(result);
+					}
+				}).run();
+			}, 500);
+			return future.wait();
+		}
+	}
+});
+
+Object.defineProperty(global.publicApi, "getApplicationPath", {
```

Comment:
these are properties and should be named as nouns, not verbs. Consider naming it `applicationPath`